web/src/lib/components/admin-page/settings/ffmpeg/ffmpeg-settings.svelte

```diff
@@ -164,6 +167,34 @@
           isEdited={config.ffmpeg.transcode !== savedConfig.ffmpeg.transcode}
         />
 
+        <SettingCheckboxes
+          label="ACCEPTED VIDEO CODECS"
+          {disabled}
+          desc="Select which video codecs should not be transcoded"
```

Comment:
```suggestion
          desc="Select which video codecs do not need to be transcoded. Only used for certain transcode policies."
```

"should not" makes it sound like it overrides other settings.